src/models/notifications/notifications.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { Controller, Post, Body } from '@nestjs/common';
+import { NotificationsService } from './notifications.service';
+import { NOTIFICATION_CREATE_SUCCESS_MSG } from 'src/common/constants/notifications';
+
+@Controller('notifications')
+export class NotificationsController {
+  constructor(private readonly notificationService: NotificationsService) {}
+
+  @Post('add')
```

Comment:
end point should be just

/notifications

POST

Reply:
API end point has been updated from `/notifications/add` to `/notifications` in latest commit